client/src/pages/ProfileListings/ProfileListings.tsx

```diff
@@ -52,31 +52,109 @@ export default function ProfileListings(): JSX.Element {
+  const initializeProfiles = () => {
+    getProfiles().then((data) => {
+      /* 
+      I've tried making the type of profileList into Profile[], but receive this error: 
+      Argument of type 'Profile | undefined' is not assignable to parameter of type 'Profile' 
+      It recognizes that the returned profile could be undefined, which would not be allowed in the array.
+      I have checks to ensure that the profile *is* defined, but the error still occurs.
+      I have also tried changing the profile interface to include undefined, to no avail.
+      */
+      const profileList: any = [];
+      const users = data.users;
+      if (users) {
+        for (let i = 0; i < users.length; i++) {
+          // Will only add 20 users to the state profiles, and make a smaller list for displayed profiles
+          if (users[i].profile) {
+            profileList.push(users[i].profile);
+            setDisplayedProfiles(profileList.slice(0, 6));
+          } else {
+            break;
+          }
+        }
+      }
+      setProfiles(profileList);
+    });
+  };
+
+  const updateDayArray = (dayArray: string[]) => {
+    setSelectedDays(dayArray);
+    const formattedDayArray: string[] = [];
+    dayArray.map((day) => {
+      day = day.charAt(0).toUpperCase() + day.slice(1);
+      formattedDayArray.push(day);
+    });
+    setDaySearchDisplay(formattedDayArray.join(', '));
+  };
+
+  const updateBySearch = async () => {
+    if (!search) {
+      initializeProfiles();
     } else {
-      setFormattedDateRange(`any`);
+      const searchResults: Profile[] = [];
+      setDisplayedProfiles([]);
+      const data = await searchProfilesByCity(search);
+      /*   
+      If I make the below users: User[], I get this error: Type 'User[] | undefined' is not assignable to type 'User[]'
+      It is recognizing that the returned data.users can include undefined entries
+      I've tried adding a check to ensure that users is not undefined, and changing the user interface to allow for undefined,
+      to no avail.
+      */
+      const users: any = data.users;
+      if (users) {
+        users.map((user: User) => {
+          if (user.profile) {
+            searchResults.push(user.profile);
+          }
+        });
+        setDisplayedProfiles(searchResults);
+      }
+    }
+  };
+
+  const updateByDays = async () => {
+    if (!selectedDays) {
+      initializeProfiles();
+    } else {
+      const searchResults: Profile[] = [];
+      const data = await searchProfilesByDay(selectedDays);
+      //   Re: any, see note above
+      const users: any = data.users;
+      if (users) {
+        users.map((user: User) => {
+          if (user.profile) {
+            searchResults.push(user.profile);
+          }
+        });
+        setDisplayedProfiles(searchResults);
+      }
     }
   };
 
   const profileCardGrid = (
     <Grid container className={classes.profilesContainer} xs={9}>
```

Comment:
Grid containers do not accept the `xs` property or any grid sizing property. Those only are for columns.